File: packages/CVMTests/scripts/python/env_test.py
#name: Python Anchors Count
#description: anchors count in html
#language: python
#environment: channels: [conda-forge, defaults], dependencies: [python=3.7, pip=22.1.2, {pip: [beautifulsoup4,]}]
#input: string html
#output: int count

from bs4 import BeautifulSoup

soup = BeautifulSoup(html, 'html.parser')
count = len(soup.findAll('a'))

File: packages/CVMTests/scripts/r/env_test.R
#name: R Env String Reverse
#description: reverse a string using stringi from a custom env
#language: r
#environment: channels: [conda-forge], dependencies: [r-stringi]
#meta.timeout: 600
#input: string input_string
#output: string reversed

library(stringi)
reversed <- stri_reverse(input_string)
